book/src/dev/rfcs/0003-state-updates.md

```diff
@@ -169,9 +169,9 @@ We use the following Sled trees:
 
 | Tree                 |                  Keys |                              Values |
 |----------------------|-----------------------|-------------------------------------|
-| `blocks_by_hash`     | `BlockHeaderHash`     | `Block`                             |
-| `hash_by_height`     | `BE32(height)`        | `BlockHeaderHash`                   |
-| `tx_by_hash`         | `TransactionHash`     | `BlockHeaderHash || BE32(tx_index)` |
+| `blocks_by_hash`     | `block::Hash`     | `Block`                             |
+| `hash_by_height`     | `BE32(height)`        | `block::Hash`                   |
+| `tx_by_hash`         | `transaction::Hash`     | `block::Hash || BE32(tx_index)` |
```

Comment:
Oops, the spacing here is weird. Not that it matters that much.

Here is a correctly spaced version using a monospace font - unfortunately GitHub uses a proportional font for suggestions.
```suggestion
| `blocks_by_hash`     | `block::Hash`         | `Block`                         |
| `hash_by_height`     | `BE32(height)`        | `block::Hash`                   |
| `tx_by_hash`         | `transaction::Hash`   | `block::Hash || BE32(tx_index)` |```

Reply:
Fixed in 6b8a5d7b